src/encoder/mod.rs

```diff
@@ -1,4 +1,4 @@
-use futures::{Stream, StreamExt, lock::Mutex,  future::join_all};
```

Comment:
cargo make reorders these.

Reply:
do not run `cargo make` in this fork.

revert all of these changes.

forks should stay as close to their upstream as possible.